### DISCUSSION TITLE
Fundamentals.rst mentions `dag_id` in Instantiate Dag Section but not shown

### DISCUSSION THREAD

**Discussion by datatalking** [apache/airflow#28298] [General]
### What do you see as an issue?

This is my first errata for the docs and under the 'Instantiate a DAG' section it mentions [(https://github.com/apache/airflow/tree/main/docs/apache-airflow/tutorial)/fundamentals.rst](https://github.com/apache/airflow/blob/main/docs/apache-airflow/tutorial/fundamentals.rst#instantiate-a-dag)

```
We'll need a DAG object to nest our tasks into. Here we pass a string that defines the `dag_id`, which serves as a unique identifier for your DAG. We also pass the default argument dictionary that we just defined and define a schedule of 1 day for the DAG
```

It mentions the `dag_id` but never shows it or mentions it again until the bottom section under Tests.

Should it be declared or used somehow?

I'm doing a fresh install on MacOs and there are so many little flags and issues I'm looking for a more thorough install. I bought the `Data Engineering with Python` book from PacktPub from Paul Crickard and so much has changed the book is becoming less and less helpful.

### Solving the problem

Should the `dag_id` object or whatever it is be declared, referenced, shown what it is or how can we be verbose on what it is? A search on that page 'https://airflow.apache.org/docs/apache-airflow/stable/tutorial/fundamentals.html' shows only two mentions of `dag_id` SO I figured this was an oversight and I'm willing to help fill in or be verbose in explaining it.

### Anything else

Once I know the purpose or have context on what `dag_id` is or can be explained as I'm willing to submit a PR.

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
The `dag_id` is unique identifier of the dag. That's all there is. I am not sure if there is anything else to be explained. If you want definition of unique identifier, here it is https://en.wikipedia.org/wiki/Unique_identifier 

And if you have better idea how to explain it in the docs - feel free to open PR. It's easy - just click "suggest a change on this page" button and you will get a PR opened and you will be able to make a proposal directly there.

BTW. This is not an issue,  but more of a question - please refrain from opening issues, instead open a discussion in the future. I will convert it into discussion now.
